src/events/event_handler.ts

```diff
@@ -127,6 +127,46 @@ export class eventHandler {
 		this.handleChanges(filePath, currentText);
 	}
 
+	handleTabChangeEvent = (tab_data: object) => {
+		/*
+		The flow for handler will be as follows:
+			- For the current open repoPath, get the repo_id and from config File
+			- Get Current Open Tabs TabGroups.ALL
+			- For each tab:
+				* Get the Relative Path of each file using repoPath
+				* Get the fileID using relative_path found in above step
+		*/
+
+		// For the current open repoPath, get the repo_id and from config File
+		const configJSON = readYML(this.settings.CONFIG_PATH);
+		const repo_id = configJSON.repos[this.repoPath].id;
+		console.log(`Repo ID: ${repo_id}`)
+		// Discard event if no repo_id found
+		if (!repo_id) return;
+
+		// Get list of current tabs
+		const open_tabs = vscode.window.tabGroups.all;
```

Comment:
why are we getting it again when we are already receiving as function parameter

Reply:
Yeah just noticed this whilst updating code. I have corrected it

Reply:
We will be getting tabs data here, instead of passing as a param